backend/src/test/java/sonia/meetapp/app/participants/ParticipantsServiceTest.java

```diff
@@ -39,6 +36,20 @@ void getAllParticipants() {
         assertThat(actualResult).hasSameElementsAs(expectedResult);
     }
 
+    @Test
+    void getAllParticipantsEventNotFound() {
+        List<Participant> testList = List.of(new Participant("Alex", "12", "alex@gmail.com"), new Participant("Marina", "333", "marina@gmail.com"), new Participant("Ivan", "2222", "ivan@gmail.com"));
```

Comment:
Sehr lange Zeile, besser mit umbrüchen z.B. alle new Participants in neue Zeilen.